TELCODOCS#2122: Configuring a local arbiter node (#92314)
openshift/openshift-docs: modules/ipi-install-additional-install-config-parameters.adoc

|
|Replicas sets the number of control plane nodes included as part of the {product-title} cluster.

a|
----
arbiter:

>     name: master

Reviewer: we should make sure this is called `arbiter` and not `master`

Author: Updated as per the suggestion.

----
|
|The {product-title} cluster requires a name for arbiter nodes.

a|
----
arbiter:

>     replicas: 3

Reviewer: lets set the replica count to 1 for this

Author: Updated as per the suggestion.

----
|

> |Replicas sets the number of arbiter nodes included as part of the {product-title} cluster.

Reviewer 2: ```suggestion
|The `replicas` parameter sets the number of arbiter nodes for the {product-title} cluster.
```

Author: Updated as per the suggestion.
